crates/aptos/src/common/types.rs

```diff
@@ -2370,8 +2370,16 @@ pub struct ChunkedPublishOption {
     /// Address of the `large_packages` move module for chunked publishing
     ///
     /// By default, on the module is published at `0x0e1ca3011bdd07246d4d16d909dbb2d6953a86c4735d5acf5865d962c630cce7`
-    /// on Testnet and Mainnet.  On any other network, you will need to first publish it from the framework
+    /// on Testnet and Mainnet. On any other network, you will need to first publish it from the framework
     /// under move-examples/large_packages.
     #[clap(long, default_value = LARGE_PACKAGES_MODULE_ADDRESS, value_parser = crate::common::types::load_account_arg)]
     pub(crate) large_packages_module_address: AccountAddress,
+
+    /// Size of the code chunk in bytes for splitting bytecode and metadata of large packages
+    ///
+    /// By default, the chunk size is set to `CHUNK_SIZE_IN_BYTES`. A smaller chunk size will result
+    /// in more transactions required to publish a package, while a larger chunk size might cause
+    /// transaction to fail due to exceeding the execution gas limit.
+    #[clap(long, default_value = CHUNK_SIZE_IN_BYTES, value_parser = clap::value_parser!(usize))]
```

Comment:
Try using just default_value_t, then you won't need the string default or the parsing function.

Reply:
Thanks for the suggestion! fixed it